corehq/apps/export/models/new.py

```diff
@@ -975,6 +975,7 @@ class InferredSchema(Document):
     group_schemas = SchemaListProperty(InferredExportGroupSchema)
     case_type = StringProperty(required=True)
     version = IntegerProperty(default=1)
+    last_app_versions = DictProperty()
```

Comment:
in the merge schemas function it expects the schema interface to have this property (it will error without it). the other option is to add an if statement in the merge logic. i think it's ok to add because semantically it makes sense that this is an empty dictionary and within the merge function the if statement would basically be `if not last_app_versions: return {}` so it can continue with the merge